app/models/user.rb

```diff
@@ -1,2 +1,7 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
```

Comment:
- [ ] The user not only has posts he also has `comments`, and `likes` as well. Kindly add these two alongside posts.